The custom precompiled-headers handling causes dependency
loops in Ninja. I don't see much value in the PCH here, so
just zero them out. (Neither do I understand how ninja ends
up with the dependency loop; make handles this without complaint)

--- CMakeLists.txt.orig	2020-01-25 12:18:16 UTC
+++ CMakeLists.txt
@@ -750,8 +750,8 @@ precompiled_header(QT_INCLUDES all ${BUILD_PCH})
 # MSVC does not require these targets, as all.h is not copied and the
 #   PCH generation is done per-project
 if (NOT MSVC)
-   ADD_CUSTOM_TARGET(mops1 DEPENDS ${PROJECT_BINARY_DIR}/all.h)
-   ADD_CUSTOM_TARGET(mops2 DEPENDS ${PCH})
+   ADD_CUSTOM_TARGET(mops1 DEPENDS ) # ${PROJECT_BINARY_DIR}/all.h)
+   ADD_CUSTOM_TARGET(mops2 DEPENDS ) # ${PCH})
 endif (NOT MSVC)
 
 ##
